src/libs/API/parameters/SyncPolicyToIntegratedConnectionParams.ts

```diff
@@ -0,0 +1,6 @@
+type SyncPolicyToIntegratedConnectionParams = {
+    policyID: string;
+    idempotencyKey: string;
+};
+
+export default SyncPolicyToIntegratedConnectionParams;
```

Comment:
I'm a little bit confused about this because this is not a command that exists on the backend.

I see that we use this as the parameter type for both of the following commands

- `READ_COMMANDS.SYNC_POLICY_TO_QUICKBOOKS_ONLINE`
- `READ_COMMANDS.SYNC_POLICY_TO_XERO`

Are there other param types defined in the `parameters` directly that is not part of the APIs?

Reply:
should we create a separate file for each command? I think the pattern we use is a one type definition for each command.  It's possible that the parameters for these commands to diverge in the future 😄

Reply:
cc @ShridharGoel